build/config.props

```diff
@@ -19,12 +19,12 @@
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.1</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.2</ReleaseLabel>
```

Comment:
When these values need to be increased? How is it tracked that the updates of these values is not missed?

Reply:
Are there partner teams that need to be aware of these changes?

Reply:
The dates are on the release schedule. Partners have access to the same schedule.

Reply:
No partner teams need to be aware of this, it's based on a schedule.
1. In VS, we insert automatically, everyone just picks it up. 
1. In .NET Core we insert automatically and the tooling picks it up. 

As far as ensuring that we don't forget it...we have issue templates for that, like https://github.com/NuGet/Client.Engineering/issues/179 for example.